src/libs/theme.js

```diff
@@ -187,11 +187,10 @@ export const Styles = {
       fontFamily: 'Montserrat',
     },
     CARDCONTAINER: {
-      marginTop: '2rem',
+      margin: '-.75rem -12.5%',
```

Comment:
```suggestion
      margin: '-.75rem -12.5% 2rem',
```

I think it's still worth visually separating the opened table from the row **below** it. So I'd recommend something like this.